### ISSUE TITLE
Avatar-manager package install breaks any links on the page(login link, or drop down list link ) 

### ISSUE THREAD

**Issue by tomvolek** [selaias/avatar-manager#7]
Hi:  I have an app with the following packages . On top of my main app page, I have several links such as drop down list and a login link.  As soon as I install avatar-manager package , all links stop working.  any clue why ?  As a result  I have to remove the avatar-manager package  all together. 

meteor update
This project is already at Meteor 1.2.1, the latest release.

meteor list
accounts-password                             1.1.4  Password support for accounts
alanning:roles                                1.2.14  Authorization package for Meteor
aldeed:collection2                            2.8.0  Automatic validation of insert and update operations on...
anti:i18n                                     0.4.3  Internalization: simplest package
aslagle:reactive-table                        0.8.21  A reactive table designed for Meteor
blaze-html-templates                          1.0.1  Compile HTML templates into reactive UI with Meteor Blaze
cfs:standard-packages                         0.5.9  Filesystem for Meteor, collectionFS
check                                         1.1.0  Check whether a value matches a pattern
ejson                                         1.0.7  Extended and Extensible JSON library
email                                         1.0.8  Send email messages
facts                                         1.0.5  Publish internal app statistics
fortawesome:fontawesome                       4.5.0  Font Awesome (official): 500+ scalable vector icons, cu...
gadicohen:messageformat                       0.0.52  MessageFormat support, the Meteor way
houston:admin                                 2.0.6  A zero-config Meteor Admin
ian:accounts-ui-bootstrap-3                   1.2.84  Bootstrap-styled accounts-ui with multi-language support.
iron:router                                   1.0.12  Routing specifically designed for Meteor
jquery                                        1.11.4  Manipulate the DOM using CSS selectors
logging                                       1.0.8  Logging facility.
meteor-base                                   1.0.1  Packages that every Meteor app needs
meteorhacks:kadira                            2.26.3  Performance Monitoring for Meteor
mizzao:bootstrap-3                            3.3.1_1  HTML, CSS, and JS framework for developing responsive...
mizzao:timesync                               0.3.4  NTP-style time synchronization between server and client
mizzao:user-status                            0.6.6  User connection and idle state tracking for Meteor
mobile-experience                             1.0.1  Packages for a great mobile user experience
momentjs:moment                               2.10.6  Moment.js (official): parse, validate, manipulate, and...
mongo                                         1.1.3  Adaptor for using MongoDB and Minimongo over DDP
multiply:iron-router-progress                 1.0.2  Progressbar for iron:router
phowat:i18n-ui-dropdown-bootstrap3-component  0.0.1  Replacement for tap-i18n-ui's i18n_dropdown as a Bootst...
rajit:bootstrap3-datepicker                   1.5.0  Meteor packaging of eternicode/bootstrap-datepicker for...
rajit:bootstrap3-datepicker-fa                1.5.0  Meteor packaging of Persian translation of eternicode/b...
random                                        1.0.5  Random number generator and utilities
reactive-var                                  1.0.6  Reactive variable
reload                                        1.1.4  Reload the page while preserving application state.
rzymek:moment-locale-fa                       2.9.0  Moment.js fa locale. Companion package for rzymek:moment
selaias:avatar-manager                        0.3.2  Manage your avatar. Upload an imag or select one from t...
session                                       1.1.1  Session variable
showdown                                      1.0.5  Moved to the 'markdown' package
spacebars                                     1.0.7  Handlebars-like template language for Meteor
standard-minifiers                            1.0.2  Standard minifiers used with Meteor apps by default.
tap:i18n                                      1.7.0  A comprehensive internationalization solution for Meteor
tap:i18n-db                                   0.4.0  Internationalization for Meteor Collections
tap:i18n-ui                                   0.7.0  User interface for the tap-i18n package
telerik:kendo-ui-core-bootstrap-theme         2015.2.805  Kendo UI Core is the free & open-source version of...
tomi:upload-jquery                            2.2.1  Client template for uploads using "jquery-file-upload" ...
tomi:upload-server                            1.3.3  Upload server for Meteor. Allows to save and serve file...
tracker                                       1.0.9  Dependency tracker to allow reactive callbacks
utilities:avatar                              0.9.2  Consolidated user avatar template (twitter, facebook, g...
wenzhixin:bootstrap-table                     1.9.0  Bootstrap table with radio, checkbox, sort, pagination,...
yuukan:streamy                                1.3.0  Simple interface to use the underlying sockjs in a mete...

**Comment by selaias** [comment]
sorry about that. I'm going to take a look. what kind of errors do you get?

**Comment by tomvolek** [comment]
Hi selaias :  amazingly no error message on the browser console or on the server side. Is there anything else I can provide you ? I am sure other users might see this issue.  all my packages are updated to the latest. 

At the top of my page I have a nav bar , inside the nav bar I have the Meteor Account login button link.  Once I install your package , the login link stops working :  Bellow the template and my css. 

<template name="navigation">

```
<nav class="navbar navbar-custom navbar-static-top" style="margin-bottom: 0px; font-size: 12px; height: 40px;">
    <div class="navbar-header">
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
        </button>

        <div id="loginButtons"> {{> loginButtons}} </div>
    </div>
    <div class="collapse navbar-collapse" >
        <div id="welcome">{{_ "welcome"}}</div>
        <ul class="nav navbar-nav">
            <li class="active"><a href="{{pathFor route='home'}}">{{_"Home"}}</a></li>
            <li> <a href="{{pathFor route='login'}}">{{_"Login"}}</a></li>
            <li><a href="{{pathFor route='register'}}">{{_"Register"}}</a></li>
            <li> <a href="{{pathFor route='auction_list'}}">{{_"Auction List"}}</a></li>
            {{#if isInRole 'admin'}}
                <!-- Reports  -->
                <li> <a href="{{pathFor route='report'}}">{{_"Report"}}</a></li>
                <li>   <a href="{{pathFor route='form_enter_product'}}">{{_"Edit/Search Items"}}</a></li>
                <li>   <a href="{{pathFor route='form_edit_user'}}">{{_"Edit Users"}}</a></li>
            {{/if}}
        </ul>
        <ul class="nav navbar-nav navbar-right">
            <div id="languages"> {{> i18n_dropdown_bootstrap3_component}} </div>
        </ul>
    </div><!-- /.navbar-collapse -->
</nav>
```

</template>
================ CSS ===============

body {
    background-color: #C3E8C1;
    margin: 0px;
    padding: 0px;
}

.tftable {font-size:10px;color:#333333;width:100%;border-width: 2px;border-color: #9dcc7a;border-collapse: collapse;}
.tftable th {font-size:10px;background-color:#abd28e;border-width: 1px;padding: 8px;border-style: solid;border-color: #9dcc7a;text-align:center;  }
.tftable tr {background-color:#ffffff;}
.tftable td {font-size:10px;border-width: 1px;padding: 8px;border-style: solid;border-color: #9dcc7a;}
.tftable tr:hover {background-color:#ffff99;}
.usertable {font-size:10px;color:#333333;width:100%;border-width: 2px;border-color: #9dcc7a;border-collapse: collapse;}
.usertable th {font-size:10px;background-color:#abd28e;border-width: 1px;padding: 8px;border-style: solid;border-color: #9dcc7a;text-align:center;  }
.usertable tr {background-color:#ffffff;}
.usertable td {font-size:10px;border-width: 1px;padding: 8px;border-style: solid;border-color: #9dcc7a; }
.usertable tr:hover {background-color:#ffff99;}

#user_field  {font-size:10px;border-width: 1px;padding: 8px;border-style: solid;border-color: #9dcc7a; width: 150px;}

div#myCanvas{ position: absolute; margin-left: 5px; }

button.buy_btn { background: red; text-align  : center; width: auto; padding-left:10px; padding-right:10px; position: absolute; z-index:1; left :700px; top:500px; font-size: 24px; }
button.inc_btn { background: red;  z-index:1; font-size: 24px; }

#Search_Item {  text-align  : center; width: auto; padding-left:10px; padding-right:10px;  z-index:1; font-size: 16px; }
#Edit_Item { text-align  : center; width: auto; padding-left:10px; padding-right:10px;    z-index:1; font-size: 16px; }
#Add_Item {  text-align  : center; width: auto; padding-left:10px; padding-right:10px;    z-index:1; font-size: 16px; }

#loginButtons {float: left;  color: #FFFFFF; }
#welcome {float: left;  margin-left: 10px; }
#languages {position: absolute; left :735px; width: 80px; top:5px;}

table {
  border: 0;
}

th {
  border: 0;
  font-size: 10px;
  text-align: center;

}

td {
  border: 1px solid #999;
  padding: 0.5rem;
  text-align: center;
  background-color: white;
}

.heading1 {height: 20px; }

.navbar-custom {
    background-color:#8AE65C;
    color:#ffffff;
    border-radius:0;
}

.navbar-custom .navbar-nav > li > a {
    color:#fff;
    padding-left:20px;
    padding-right:20px;
}
.navbar-custom .navbar-nav > .active > a, .navbar-nav > .active > a:hover, .navbar-nav > .active > a:focus {
    color: #ffffff;
    background-color:transparent;
}

.navbar-custom .navbar-nav > li > a:hover, .nav > li > a:focus {
    text-decoration: none;
    background-color: #33aa33;
}

.navbar-custom .navbar-brand {
    color:#eeeeee;
}
.navbar-custom .navbar-toggle {
    background-color:#eeeeee;
}
.navbar-custom .icon-bar {
    background-color:#33aa33;

}

.boxed {
    text-align:left;
    text-shadow: 0 0 5px blue;
}

**Comment by selaias** [comment]
can you post also the translations file?

I have copied your package list into an empty project and everything seems ok.

**Comment by tomvolek** [comment]
HI Selaias :  

if you look at the HTML file above , the two link which do not fire are : 

<div id="loginButtons"> {{> loginButtons}} </div>

<div id="languages"> {{> i18n_dropdown_bootstrap3_component}} </div>

 Just a note, if I remove your package , each string in drop down link gets translated correctly.  for some reason it seems the click event gets blocked all together. It behaves the same way on chrome or Safari or Firefox.  

Is this what you mean by translation file ? 
 "Supplier Name":"نام تولید کننده",
 "ID":"کد",
 "Country":"کشور",
 "Grade":"درجه",
 "Monetary Unit":"واحد پولی",
 "Price/Flower":"قیمت / گل",
 "Buyer Number":"شماره خریدار",
 "Trolleys":"کانتینر" ,
 "Containers/Trolley":"کانتینر/سطل ",
 "Total Containers":"مجموع کانتینر",
 "Flower/Container":"گل /سطل ",
 "Minimum Container To Buy":"حداقل کانتینر برای خرید",
 "Admin Section Current User ":"بخش مدیریت، کاربر جاری",
 "Set Item for Auction ": "بفرش بذارید",
 "Product ID":"کد محصول",
 "Set Auction":"حراج",
 "Search/Add products":"جستجو / اضافه کردن محصولات",
 "Purchased Report": "گزارش خرید",
 "Search":"جستجو",
 "Edit":"ویرایش",
 "Add":"اضافه کن",
"Producer Name":"نام سازنده",
"Producer ID":"کد سازنده",
"Product Name":"نام محصول",
"Product Image":"تصویر محصول",
"NumberOfItems":"تعداد آیتم ها",
"PricePerFlower":"قیمت هر گل",
"You purchased  containers":"اقلام خریداری شد",
"Product_grade":"درجه",
"Welcome to Auction":"به حراج خوش آمدید",
"Home":"خانه",
"Login":"وارد شدن",
"Register" : "ثبت نام",
"Auction List" : "فهرست حراج",
"Report" : "گزارش",
"Edit/Search Items" : "ویرایش / جستجو",
"Edit Users" : "اضافه کردن / ویرایش کاربر",
"Payaneh Auction" : "حراج پایانه",
"Browse": "فهرست",
"dropFiles" : "رها کردن فایل ها در اینجا",
"browse": "فهرست",
"cancelled": "لغو شد",
"remove": "برداشتن",
"upload": "آپلود",
"done": "انجام شده",
"cancel": "لغو",
"Created At":"ایجاد شده در",
"Creator Id":"خالق ها",
"Auction Date":"تاریخ حراج",
"AuctionDate": "تاریخ حراج",
"Drop Files Here":"رها کردن فایل ها در اینجا",
"There are currently":"در حال حاضر",
"users in Auction":"کاربران در حراج"

**Comment by selaias** [comment]
the problem is with the bootstrap package you are using. according to the [https://github.com/ianmartorell/meteor-accounts-ui-bootstrap-3] package you are using the `twb:bootstrap` is the recommended package and you are using the `mizzao:bootstrap-3`. 
The `avatar-manager` is using already the official `twb:bootstrap` package and somehow there is a confict. if you remove the `mizzao:bootstrap-3` package it will solve your problem.

**Comment by tomvolek** [comment]
Hi selaias.  That was the problem. Thanks very much for looking into this .  Great package 

Best regards 
Tom

**Comment by selaias** [comment]
Glad we sorted it out. Thanks
closing this

**Comment by tomvolek** [comment]
Selaias how long you been working with meteor ? What is your job ?  I live in Silicon Valley , San Francisco work for a company we do call center software. 

I discovered it 6 month ago and love it .  I wanted to write an auction app which price increases are sent to clients real time. I had to use streamy package to make sure I have real time broadcast of clients.   It is working very nicely for certain number of  clients . My target is around 200 clients max. 

I have to do some research to see how I can scale meteor from one local server to a pool of them. How would I handle load balancing etc.  


Tom

**Comment by tomvolek** [comment]
Hi Elaias : 
   I added {{ >avatarManager }} to my html . Now I see three objects listed on HTML page which one has title of CurrentAvatar with a red top and an upload button, I try to upload an image but the save Button can not be pressed to save .  Is there a specific format or size which the image needs to be ?     
The other two objects showing up are Gavatar and Default avatars boxes. How can I get ride of them?    
Thanks

```
On Sunday, January 3, 2016 11:07 AM, Stavros Elaias <notifications@github.com> wrote:
```

 Glad we sorted it out. Thanks
closing this—
